### ISSUE TITLE
[BUG] cannot create launcher icons on web

### ISSUE THREAD

**Issue by piratkin** [mrrhak/icons_launcher#22]
I'm trying to create a favicon for the web version and I'm getting the following error:

```bash
$ flutter pub run icons_launcher:create --path icons_launcher.yaml


╔════════════════════════════════════════════════════╗
║             ✨✨  ICONS LAUNCHER  ✨✨             ║
╠════════════════════════════════════════════════════╣
║                   Version: 2.0.3                   ║
╚════════════════════════════════════════════════════╝


Unhandled exception:
NoSuchMethodError: The method 'containsKey' was called on null.
Receiver: null
Tried calling: containsKey("adaptive_background_color")
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:38:5)
#1      _createIconsByConfig (package:icons_launcher/cli_commands.dart:332:28)
#2      createLauncherIcons (package:icons_launcher/cli_commands.dart:29:3)
#3      main (file:///C:/Users/user/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/icons_launcher-2.0.3/bin/create.dart:15:3)
#4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:32)
#5      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
pub finished with exit code 255
```

This is my config file:

```bash
$ cat icons_launcher.yaml
# flutter pub run icons_launcher:create

icons_launcher:
  image_path: 'assets/images/favicon.png'
  platforms:
    web:
      enable: true
      image_path: 'assets/images/favicon.png'
      favicon_path: 'assets/images/favicon.png'
```

flutter doctor

```bash
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel master, 3.1.0-0.0.pre.2294, on Microsoft Windows [Version 10.0.17763.2237], locale en-UA)
[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc4)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.1.4)
[√] Android Studio (version 4.2)
[√] Connected device (2 available)
[√] HTTP Host Availability

• No issues found!
```

**Comment by mrrhak** [comment]
Hello, @piratkin 
Thank you for reporting. Now I have fixed this issue with the released [v2.0.4](https://pub.dev/packages/icons_launcher).

**Comment by piratkin** [comment]
You are right, it is. I updated and everything works.

Is it possible to make the background transparent?
